x-pack/legacy/plugins/siem/public/components/resize_handle/index.tsx

```diff
@@ -20,28 +20,43 @@ export const calculateDeltaX = ({ prevX, screenX }: { prevX: number; screenX: nu
 
 const isSafari = /^((?!chrome|android|crios|fxios|Firefox).)*safari/i.test(navigator.userAgent);
 
-interface Props {
-  /** the `onResize` callback will be invoked with this id */
-  id: string;
-  /** The resizeable content to render */
-  render: (isResizing: boolean) => React.ReactNode;
-  /** a (styled) resize handle */
-  handle: React.ReactNode;
+interface ResizeHandleContainerProps {
+  bottom?: string | number;
   /** optionally provide a height style ResizeHandleContainer */
   height?: string;
+  left?: string | number;
+  positionAbsolute?: boolean;
+  right?: string | number;
+  top?: string | number;
+}
+
+interface Props extends ResizeHandleContainerProps {
+  /** a (styled) resize handle */
+  handle: React.ReactNode;
+  /** the `onResize` callback will be invoked with this id */
+  id: string;
   /** invoked when the handle is resized */
   onResize: OnResize;
+  /** The resizeable content to render */
+  render: (isResizing: boolean) => React.ReactNode;
 }
 
 interface State {
   isResizing: boolean;
 }
 
-const ResizeHandleContainer = styled.div<{ height?: string }>`
-  cursor: ${resizeCursorStyle};
-  ${({ height }) => (height != null ? `height: ${height}` : '')}
+const ResizeHandleContainer = styled.div<ResizeHandleContainerProps>`
+  ${({ bottom, height, left, positionAbsolute, right, theme, top }) => css`
+    bottom: ${positionAbsolute && bottom};
```

Comment:
if `positionAbsolute` is falsy, the `&&` will short-circuit, resulting in:

```
bottom:;
```

consider replacing the `&&` with an expression that doesn't omit a `bottom` attribute if `positionAbsolute` is not truthy, like:

```
${positionAbsolute ? `bottom: ${bottom};` : ''};
```

(this comment is also applicable to `left`, `position`, `right`, `top`, and `z-index` in this code block)